extensions/amp-story/1.0/media-tasks.js

```diff
@@ -395,6 +419,7 @@ export class SwapIntoDomTask extends MediaTask {
 
     copyCssClasses(this.placeholderEl_, mediaEl);
     copyAttributes(this.placeholderEl_, mediaEl);
+    rotateWhitelistedProperties(this.placeholderEl_, mediaEl);
```

Comment:
Why are you doing this? Is this guaranteed to point to the same media?

Reply:
> Why are you doing this?

There are restrictions on the number of `<video>` you can have on iOS, after a certain number of elements, they just don't play.
To solve this for Stories, the "Media pool" creates a small number of `<video>` elements and rotates through them. It updates its sources from the original video, and swaps it in the DOM.
This line ensures that, as the origin video is swapped for one from the media pool, it keeps the error property.

> Is this guaranteed to point to the same media?

Yes

Reply:
If we change the src of a video element, does it try to redownload the video? If so, we've reset the load.

Reply:
Good point. Cf comment on the main PR thread, I changed a few things